src/tools/rbd_wnbd/rbd_wnbd.cc

```diff
@@ -407,11 +407,11 @@ int map_device_using_suprocess(std::string arguments, int timeout_ms)
 
   dout(5) << __func__ << ": command line: " << command_line.str() << dendl;
 
-  GetStartupInfo(&si);
+  GetStartupInfoW(&si);
```

Comment:
Just curious, why `si` was initialized with `GetStartupInfo()` here but not in `src/common/win32/SubProcess.cc`?

Reply:
This call was intended to reuse the stdin/stdout/stderr handles of the parent process, unlike ``src/common/win32/SubProcess.cc``.